layouts/partials/contact-us.html

```diff
@@ -7,17 +7,17 @@ <h3>Ask Us About</h3>
                 <ul>
                     <li>
                         How to deliver
-                        <a href="{{ relref . "/why-pulumi/delivering-cloud-native-infrastructure-as-code" }}">
+                        <a data-tracking-id="partials-contact-us-why-pulumi" href="{{ relref . "/why-pulumi/delivering-cloud-native-infrastructure-as-code" }}">
                             cloud native infrastructure as code</a>, with general purpose programming languages.
                     </li>
                     <li>
                         How Pulumi supports infrastructure, managed services,
-                        <a href="{{ relref . "/topics/containers.md" }}">containers</a>,
-                        <a href="{{ relref . "/topics/kubernetes.md" }}">Kubernetes</a>, and
-                        <a href="{{ relref . "/topics/serverless.md" }}">serverless</a> infrastructure and apps.
+                        <a data-tracking-id="partials-contact-us-containers" href="{{ relref . "/topics/containers.md" }}">containers</a>,
```

Comment:
Not a big deal, but just as a head's up, Justin's got a PR (#2196) to remove all `.md` extensions, so you might want to rebase on that, and replace-out the ones in the files you're touching as well, in case Git gets confused.

Reply:
Ah, sorry about the merge conflicts @zchase :(